### ISSUE TITLE
[勘误]`张量操作`和`神经网络操作`的区别

### ISSUE THREAD

**Issue by hbsun2113** [openmlsys/openmlsys-zh#397]
在[4.2. 计算图的基本构成](https://openmlsys.github.io/chapter_computational_graph/components_of_computational_graph.html#:~:text=%E5%BC%A0%E9%87%8F%E6%93%8D%E4%BD%9C%EF%BC%9A%E5%8C%85%E6%8B%AC,Moment%20Estimation%2C%20Adam)%E7%AD%89%E3%80%82)中将`张量操作`和`神经网络操作`分为两个概念，但从所举的例子来看`神经网络操作`都是`张量操作`。请问为什么两者的区别是什么，为什么要分为两个概念？
![image](https://user-images.githubusercontent.com/27994442/182107017-a11ce192-5e25-42f5-ae47-c55d4a661765.png)

**Comment by hanjr92** [comment]
你好，这个部分的操作区别主要还是有区别的，张量算子更多是对tensor本身的shape、dim、dtype以及求取特征值基础的操作，而神经网络算子是相对高阶的过程还是有别于张量操作算子的，比如矩阵乘法和卷积运算可能会认为都有矩阵相乘过程的参与，但是卷积的过程更加复杂，卷积核与特征图之间卷积运算还需要考虑特征图的padding扩充、卷积核的步长、卷积核的空洞卷积等。对应到深度学习框架里面 张量的操作更像是 ms.ops.matmul/torch.tensor.matmul 而神经网络算子对应于 ms.nn.conv/torch.nn.functional.conv，这样的分类也避免了将大量的内容集中在同一概念下。
